sim/src/main/cc/bridges/TracerVModule.cc

```diff
@@ -10,10 +10,10 @@
 #include <iostream>
 #include <string_view>
 
-static std::vector<bool> get_contiguous(unsigned bits, unsigned total) {
+static std::vector<bool> get_valids(unsigned seed, unsigned total) {
   std::vector<bool> ret;
   for (unsigned i = 0; i < total; i++) {
-    const bool value = (i < bits);
+    const bool value = seed & (1 << i);
```

Comment:
So is it true that this will only produce inputs with a single valid instruction or am i misunderstanding? Could you not just call out to an RNG at this point?

Reply:
No, this code sweeps a 1 bit mask through seed. This is the core mechanic to turn a uint into a vector of bools.

Reply:
Ok right.  It's a shame there's not a convenient one liner for this. Maybe you want an assert to check the requested total is less than the bitwidth of the seed.